Source/WarpX.H

```diff
@@ -760,9 +769,19 @@ private:
         return gather_buffer_masks[lev].get();
     }
 
-    void ReorderFornbergCoefficients(amrex::Vector<amrex::Real>& ordered_coeffs,
-                                     amrex::Vector<amrex::Real>& unordered_coeffs,
-                                     const int order);
+#ifdef WARPX_USE_PSATD
+    // TODO Add Doxygen
```

Comment:
Could you also add the docstrings in this file?

Reply:
Yes, done in https://github.com/ECP-WarpX/WarpX/pull/1763/commits/47a930ac4da229cb1fa41e61e1211fd8fd297172.